ec/ecresource/deploymentresource/resource.go

```diff
@@ -15,24 +15,23 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package ecresource
+package deploymentresource
```

Comment:
Looking at other providers (even though they don't have their resources separated into packages like we do) the naming for their functions is like this: [`resourceLocalFIle`](ec/ecresource/deploymentresource/resource.go) or `resourceAwsAmi`. Going forward I think it would make sense to follow this naming shceme and rename this package to `resourcedeployment`. I know it might look weird, but I _think_ we need to be somewhat consistent with the other providers? what do you think? 🤔

Reply:
I don't have any strong opinions on naming here, if that's what other providers do, we can easily do that

Reply:
I don't have super strong opinions either, but just wondering if this might be something that they'll insist on eventually 😄 I've left the repo approved in any case, so you can either change it or not